### PR TITLE
Added Hindi translation

### PR DESCRIPTION
**Pull request by pratik1424**

### CODE REVIEW

File: src/main/resources/assets/purpurclient/lang/hi_in.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "purpurclient.options.fix-chat-stutter":  "ऐसिंक चैट",
+    "purpurclient.options.fix-chat-stutter.tooltip": "सक्षम एक अन्य थ्रेड पर चैट को संसाधित करेगा ताकि यह रेंडर थ्रेड को अवरुद्ध न हो",
+    "purpurclient.options.bee-count-in-debug": "मधुमक्खी गिनती",
+    "purpurclient.options.bee-count-in-debug.tooltip": "सक्षम ब्लीडिंग स्क्रीन में मधुमक्खियों की गिनती जोड़ देगा (F3) जब मधुमक्खियों के कुत्ते और घोंसले देखते हैं",
+    "purpurclient.options.splash-screen": "लेश स्क्रीन",
+    "purpurclient.options.splash-screen.tooltip": "आधारित करने के लिए, Mojang के बजाय Purpur की स्प्रैच स्क्रीन का उपयोग करेगा",
+    "purpurclient.options.window-title": "खिड़की का नाम",
+    "purpurclient.options.window-title.tooltip": "सक्षम करें, Mojang के बजाय Purpur का खिड़की शीर्षक का उपयोग करेगा",
+    "purpurclient.options.mobs": "मोब्स",
+    "purpurclient.options.mobs.tooltip": "Open mob options स्क्रीन",
```
Comment (RealRONiN) [inline src/main/resources/assets/purpurclient/lang/hi_in.json:13]: Replace with "मोब्स विकल्प स्क्रीन खोलें".

File: src/main/resources/assets/purpurclient/lang/hi_in.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  
+    "purpurclient.title.singleplayer": "%s पर एकल खिलाड़ी",
```
Comment (RealRONiN) [inline src/main/resources/assets/purpurclient/lang/hi_in.json:15]: Replace with "सिंगलप्लेयर पर %s".

File: src/main/resources/assets/purpurclient/lang/hi_in.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "purpurclient.title.multiplayer.realms": "मल्टीप्लेयर (%s रेलमस पर )",
+    "purpurclient.title.multiplayer.lan": "मल्टीप्लेयर (%s LAN पर )",
```
Comment (RealRONiN) [inline src/main/resources/assets/purpurclient/lang/hi_in.json:17]: Repalce "LAN" with "लैन".

File: src/main/resources/assets/purpurclient/lang/hi_in.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    "purpurclient.title.multiplayer.server": "मल्टीप्लेयर (%s पर %s)",
+    "purpurclient.title.multiplayer.unknown": "मल्टीप्लेयर (%s पर 3rd-party सर्वर)",
+
+    "purpurclient.options.on": "%s: \u00a7aON",
```
Comment (RealRONiN) [inline src/main/resources/assets/purpurclient/lang/hi_in.json:21]: Replace "ON" with "चालू".

**Review by RealRONiN — CHANGES_REQUESTED** [review]
Should've used Google Translate, Hindi is my second language so I know that requested changes are required to get the translations right.

**Comment by pratik1424** [comment]
Hey @RealRONiN,
Thank you so much for the suggestions, I have made all the necessary corrections. 
Let me know if you find more.

**Review by RealRONiN — APPROVED** [review]
Looks good to me.

**Comment by pratik1424** [comment]
@BillyGalbreath Any updates on this PR?

**Review by granny — APPROVED** [review]

**Pull request merged**